docs/ref/contrib/gis/geoquerysets.txt

```diff
@@ -708,11 +708,19 @@ Backend     SQL Equivalent
 ==========  ======================================
 PostGIS     ``ST_DWithin(poly, geom, 5)``
 Oracle      ``SDO_WITHIN_DISTANCE(poly, geom, 5)``
+SpatiaLite  ``PtDistWithin(poly, geom, 5)``
 ==========  ======================================
 
 .. note::
+    The default units being queried against will vary by backend. On PostGIS
```

Comment:
You might understand all this better than I do (not a GIS user myself) so feel free to amend as you see fit.